Documentation/contributor-guide/features.md

```diff
@@ -34,6 +34,7 @@ Any new enhancements to the etcd are typically added as an experimental feature.
 - Open an issue
     - It must provide a clear need for the proposed feature.
     - It should list development work items as checkboxes. There must be one work item towards future graduation to the stable future.
+    - Label the issue with `type/feature` and `experimental`.
```

Comment:
I am proposing two new labels, experimental and deprecated. They should help with tracking. cc @serathius @ahrtr